src/screens/Customers/components/RecordsTabMenu.tsx

```diff
@@ -24,23 +21,23 @@ const styles = (theme: Theme) =>
     },
   });
 
-interface TabProps {
-  classes: { tabs: string; tab: string; tabPanel: string };
-  defaultTab: string;
+interface RecordsTabMenuProps {
+  classes: { tab: string; tabPanel: string };
   invoices: MeterReadingRecord[];
   payments: PaymentRecord[];
+  defaultTab?: string;
 }
 
-function TabMenu(props: TabProps) {
+function RecordsTabMenu(props: RecordsTabMenuProps) {
   const { classes, invoices, payments, defaultTab } = props;
-  const [value, setValue] = React.useState(defaultTab);
+  const [value, setValue] = React.useState(defaultTab ? defaultTab : "0");
   const changeTab = (event: React.ChangeEvent<{}>, newValue: string) => {
     setValue(newValue);
   };
 
   return (
     <TabContext value={value}>
-      <Tabs className={classes.tabs} textColor="primary" indicatorColor="primary" value={value} onChange={changeTab}>
+      <Tabs textColor="primary" indicatorColor="primary" value={value} onChange={changeTab}>
```

Comment:
Could you add a TODO to address the empty state for this screen?

Reply:
What do you mean empty state?